src/libcore/num/mod.rs

```diff
@@ -1436,33 +1446,35 @@ macro_rules! uint_impl {
             unsafe { intrinsics::ctlz(self as $ActualT) as u32 }
```

Comment:
Shouldn't the example of leading_zeroes get the same treatment, rather than hardcoding `u16`?

Reply:
(And ditto for other methods here whose docs still list specific types in their examples.)

Reply:
Some methods were left as is because it was hard to make the example generic.

Reply:
If you can't convert all the examples, I would like to see the "examples are shared between primitive integer types" notice kept on the primitive docs, since it still applies.